src/chrome/content/rules/Bitcoin.pl.xml

```diff
@@ -1,4 +1,9 @@
+
 <!--
+Disabled by https-everywhere-checker because:
+Fetch error: http://bitcoin.pl/login => https://bitcoin.pl/login: None
```

Comment:
'None' isn't descriptive enough for me to address this. I can guess that it's detecting the mixed bug from Facebook, and if that's the case, then great! (that a bug is being blocked)